spec/lib/bamboozled/api/employee_spec.rb

```diff
@@ -137,4 +137,56 @@
       expect(employee["headers"]).to eq(expected_headers)
     end
   end
+
+  context 'when an API BASE is set' do
```

Comment:
Maybe we should be consistent with the style of the base repo (they use double quotes) or refactor and change the code style for all of the project if we're not applying these changes on the base repo